packages/vscode-extension/src/project/metro.ts

```diff
@@ -64,6 +68,7 @@ type MetroEvent =
 export class Metro implements Disposable {
   private subprocess?: ChildProcess;
   private _port = 0;
+  private _initialBundleLineOffset = 0;
```

Comment:
I don't like "initial" word here as it isn't accurate. I think the offset applies for the whole bundle not just for the first bundle that gets loaded as it is currently implemented.

We should rename "mainBundleLineOffset"